201-vm-os-disk-and-data-disk-existing-vnet/README.md

```diff
@@ -0,0 +1,24 @@
+# Create a virtual machine from two disks (OS + data disk) in an existing virtual network
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F201-os-disk-and-data-disk-existing-vnet%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-os-disk-and-data-disk-existing-vnet%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+## Prerequisites
+
+- VHD files that you want to create a VM from already exists in a storage account.
+- Name of the existing VNET and subnet you want to connect the new virtual machine to.
+- Name of the Resource Group that the VNET resides in.
+
+```
+NOTE
+
+This template will create an additional Standard_LRS storage account for enabling boot diagnostics each time you execute this template. To avoid running into storage account limits, it's best to delete the storage account when the VM is deleted.
```

Comment:
I think this is only the case if you specify a different VM name; if you re-use the same VM name, it looks like it will re-use the existing storage account